### PR TITLE
[UPDATE BONUS GUIDE] Fulcrum

### PR DESCRIPTION
**Pull request by DanielPX1**
#### What

Update Fulcrum to a latest version 1.8.2 with several small format and text improvements. 

### Why

Make new RaspiBolt users use the latest version of the software to ensure stability and previously known bugs are fixed.

#### How

Changing links for the old version to the ones pointing at the latest one. Some expected outputs had to be changed as well to match the new versions. Updated commands.

#### Scope

- [ ] Review the guide

#### Test & maintenance

By following the guide and reporting issues

### CODE REVIEW

File: guide/bonus/bitcoin/fulcrum.md [hunk 1]
```diff
@@ -199,17 +199,17 @@ $ gpg --verify Fulcrum-1.7.0-arm64-linux.tar.gz.asc
...
 * Install Fulcrum and check the correct installation requesting the version
 
 ```sh
-$ tar -xvf Fulcrum-1.7.0-arm64-linux.tar.gz
-$ sudo install -m 0755 -o root -g root -t /usr/local/bin Fulcrum-1.7.0-arm64-linux/Fulcrum Fulcrum-1.7.0-arm64-linux/FulcrumAdmin 
+$ tar -xvf Fulcrum-1.8.1-arm64-linux.tar.gz
+$ sudo install -m 0755 -o root -g root -t /usr/local/bin Fulcrum-1.8.1-arm64-linux/Fulcrum Fulcrum-1.8.1-arm64-linux/FulcrumAdmin 
 $ Fulcrum --version
-> Fulcrum 1.7.0 (Release 4ee413a)
+> Fulcrum 1.8.1 (Release 4ee413a)
```
Comment (twofaktor) [inline guide/bonus/bitcoin/fulcrum.md:212]: ->> Fulcrum 1.8.1 (Release c80bf1b)

File: guide/bonus/bitcoin/fulcrum.md [hunk 2]
```diff
@@ -199,17 +199,17 @@ $ gpg --verify Fulcrum-1.7.0-arm64-linux.tar.gz.asc
...
 compiled: gcc 7.5.0
```
Comment (twofaktor) [inline guide/bonus/bitcoin/fulcrum.md:213]: ->> compiled: gcc 8.4.0

File: guide/bonus/bitcoin/fulcrum.md [hunk 3]
```diff
@@ -351,7 +351,7 @@ Expected output:
 Jul 28 12:20:13 rasp Fulcrum[181811]: [2022-07-28 12:20:13.063] simdjson: version 0.6.0
 Jul 28 12:20:13 rasp Fulcrum[181811]: [2022-07-28 12:20:13.063] ssl: OpenSSL 1.1.1n  15 Mar 2022
 Jul 28 12:20:13 rasp Fulcrum[181811]: [2022-07-28 12:20:13.063] zmq: libzmq version: 4.3.3, cppzmq version: 4.7.1
-Jul 28 12:20:13 rasp Fulcrum[181811]: [2022-07-28 12:20:13.064] Fulcrum 1.7.0 (Release 4ee413a) - Thu Jul 28, 2022 12:20:13.064 CEST - starting up ...
+Jul 28 12:20:13 rasp Fulcrum[181811]: [2022-07-28 12:20:13.064] Fulcrum 1.8.1 (Release 4ee413a) - Thu Jul 28, 2022 12:20:13.064 CEST - starting up ...
```
Comment (twofaktor) [inline guide/bonus/bitcoin/fulcrum.md:354]: Same of line 212

**Review by twofaktor — CHANGES_REQUESTED** [review]
Compare with your results, utACK in arm64

**Comment by DanielPX1** [comment]
> Compare with your results, utACK in arm64

For some reason cant see that, no number follows after "Release" for me. Put your values in there instead.
Fulcrum 1.8.1 (Release )
compiled: gcc 8.4.0

**Review by twofaktor — APPROVED** [review]
utACK in arm64! LGTM 👍

**Comment by DanielPX1** [comment]
> utACK in arm64! LGTM 👍

Glad to hear! Can we ask for a review @VajraOfIndra @Stadicus @kristapsk ?

**Comment by twofaktor** [comment]
Change line [248](https://github.com/raspibolt/raspibolt/pull/1097/files#diff-730f9901c4e7cac9c3b645e8bbe633d5e208fa99886bbee3fc28d2c873c3b8a0R248) to:

`* Change to fulcrum data folder and generate cert and key files for SSL. Press `Enter` until the prompt is shows again`

File: guide/bonus/bitcoin/fulcrum.md [hunk 4]
```diff
@@ -417,155 +421,170 @@ This way, you can connect the BitBoxApp or Electrum wallet also remotely, or eve
...
+* Add or modify the following line. Save and exit
 
-```sh
-BTCEXP_ELECTRUM_SERVERS=tls://127.0.0.1:50002
-```
+  ```sh
+  BTCEXP_ELECTRUM_SERVERS=tls://127.0.0.1:50002
+  ```
+
+* Exit the `btcrpcexplorer` user exiting and open `btcrpcexplorer` service
```
Comment (twofaktor) [inline guide/bonus/bitcoin/fulcrum.md:466]: Change to --> * Return to `admin` user exiting and open `btcrpcexplorer` service

**Review by twofaktor — CHANGES_REQUESTED** [review]
Suggested redaction improvement

File: guide/bonus/bitcoin/fulcrum.md (hunk 4 above)
Comment (DanielPX1) [reply guide/bonus/bitcoin/fulcrum.md]: Changed to suggested text

**Comment by twofaktor** [comment]
LGTM again! tACK is ready to merge!

**Comment by DanielPX1** [comment]
Great!

File: guide/bonus/bitcoin/fulcrum.md [hunk 5]
```diff
@@ -173,240 +175,242 @@ We have our Bitcoin Core configuration file set up and now we can move to next p
...
-Jul 28 12:20:14 rasp Fulcrum[181811]: [2022-07-28 12:20:14.491] Coin: BTC
-Jul 28 12:20:14 rasp Fulcrum[181811]: [2022-07-28 12:20:14.492] Chain: main
-Jul 28 12:20:14 rasp Fulcrum[181811]: [2022-07-28 12:20:14.494] Verifying headers ...
-Jul 28 12:20:19 rasp Fulcrum[181811]: [2022-07-28 12:20:19.780] Initializing header merkle cache ...
-Jul 28 12:20:21 rasp Fulcrum[181811]: [2022-07-28 12:20:21.643] Checking tx counts ...
-...
-```
+  ```sh
+  $ sudo systemctl status fulcrum.service
+  $ sudo journalctl -fu fulcrum.service
```
Comment (twofaktor) [inline guide/bonus/bitcoin/fulcrum.md:348]: Change to -->

`sudo journalctl -f -u fulcrum`

to standardize with the rest of the guide

**Review by twofaktor — CHANGES_REQUESTED** [review]
A minor fix

File: guide/bonus/bitcoin/fulcrum.md (hunk 5 above)
Comment (DanielPX1) [reply guide/bonus/bitcoin/fulcrum.md]: done!

**Review by twofaktor — APPROVED** [review]
LGTM again! uACK!

**Comment by DanielPX1** [comment]
> LGTM again! uACK!

Great! looks ready 2 merge

**Review by twofaktor — APPROVED** [review]
tACK

**Comment by twofaktor** [comment]
> @twofaktor could you provide the version string? Sadly I cant access my node remotely, there have to be some issue

Fulcrum 1.8.2 ---> (Release d330248)

It's the only thing different

**Comment by DanielPX1** [comment]
> > @twofaktor could you provide the version string? Sadly I cant access my node remotely, there have to be some issue
> 
> Fulcrum 1.8.2 ---> (Release d330248)
> 
> It's the only thing different

Sorry I took some time, its done now

File: guide/bonus/bitcoin/fulcrum.md [hunk 6]
```diff
@@ -173,240 +175,242 @@ We have our Bitcoin Core configuration file set up and now we can move to next p
...
-[Install]
-WantedBy=multi-user.target
-```
+  ```sh
+  # RaspiBolt: systemd unit for Fulcrum
+  # /etc/systemd/system/fulcrum.service
+  
+  [Unit]
+  Description=Fulcrum
+  Wants=bitcoind.service
```
Comment (twofaktor) [inline guide/bonus/bitcoin/fulcrum.md:317]: Change to ---> `PartOf=bitcoind.service`

ref PR: #1118

**Review by twofaktor — APPROVED** [review]
A minor suggestion

File: guide/bonus/bitcoin/fulcrum.md (hunk 6 above)
Comment (DanielPX1) [reply guide/bonus/bitcoin/fulcrum.md]: Good suggestion. Added

**Review by twofaktor — APPROVED** [review]
LGTM! some maintainer would review this PR?? @VajraOfIndra please, this PR was opened a month ago and has no comments...

**Pull request merged**